pkg/draft/pack/pack_test.go

```diff
@@ -24,15 +23,19 @@ cleanup-task = "echo cleanup"
 `
 
 func TestSaveDir(t *testing.T) {
+	dockerPerm := os.FileMode(0664)
+	winDockerPerm := os.FileMode(0666)
+	tasksPerm := os.FileMode(0644)
+	winTasksPerm := os.FileMode(0666)
```

Comment:
Surely nothing that would prevent merging this PR, but I think would help if we had the permissions as constants, with expressive naming - this is something we should use across Draft (as there are also other uses of permission literals).

But not necessarily concerning this PR.

Reply:
This is in a test and not necessarily representative of what the file permissions would be for these files. Do you still think a constant would be helpful here or do you think we can leave these as variable? @radu-matei